lib/transformTwitterHandle.ts

```diff
@@ -0,0 +1,10 @@
+import * as _ from "lodash"
+
+const transformTwitterHandle = (twitterHandle?: string): string => {
+    if (twitterHandle) {
+      return _.startsWith(twitterHandle, "@", 0) ? twitterHandle : `@${twitterHandle}`
+    }
+    return "anonymous"
+}
+
+export default transformTwitterHandle
```

Comment:
can you add a new line , linter will complain

Reply:
should be fixed now? 👀